### ISSUE TITLE
FileUpload: Button not focusable using tab (Accessibility)

### ISSUE THREAD

**Issue by javiermiraticarum** [primefaces/primefaces#7836]
**Describe the defect**
Using the fileupload component with mode="simple" and skinSimple="true" the button is not focusable using the keyboard.

**Environment:**
 - PF Version: _8.0_
 - Affected browsers: _ALL_

**Expected behavior**
The button should be able to be focused by pressing the tab key.
It seems that the problem occurs because the button has the property tabindex="-1".
If it is changed to tabindex="0" the button is focusable.

**Example XHTML**
```html
<h:form id="formId">
    <p:fileUpload id="uploader3" mode="simple" skinSimple="true" styleClass="p-d-block p-mb-5" title="my custom title"/>
</h:form>
```

**Example HTML**
Current 
```html
<input type="file" id="formId:uploader3_input" name="formId:uploader3_input" tabindex="-1" aria-hidden="true" title="my custom title">
```
Expected
```html
<input type="file" id="formId:uploader3_input" name="formId:uploader3_input" tabindex="0" aria-hidden="true" title="my custom title">

or

<input type="file" id="formId:uploader3_input" name="formId:uploader3_input" aria-hidden="true" title="my custom title">
```

**Comment by melloware** [comment]
Good catch I will fix this...
